NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -160,6 +172,12 @@ def begin(
             self.__radio_sky = healpy.pixelfunc.ud_grade(self.__radio_sky, self.__n_side)
             self.__noise_temperatures[i_freq] = self.__radio_sky
 
+
+    @functools.lru_cache(maxsize=1024)
+    def get_cached_antenna_response(self, ant_pattern, zen, azi, *ant_orient):
```

Comment:
Please make this a private function (or add a docstring).

Reply:
Done (both actually)